tests/common.go

```diff
@@ -7144,6 +7145,34 @@ func CreateAzureBucket(bucketName string, objectlock bool, mode Mode, retentionD
 
 }
 
+// UpdateConfigmap creates or updates an existing configmap with the given data
+func UpdateConfigmap(configMapName string, data map[string]string) error {
+	configmap, err := k8sCore.GetConfigMap(configMapName, defaulttorpedoNamespace)
+	if k8serrors.IsNotFound(err) {
+		log.InfoD("Creating Configmap: %v", configMapName)
+		metaObj := metaV1.ObjectMeta{
+			Name:      configMapName,
+			Namespace: defaulttorpedoNamespace,
+		}
+		obj := &corev1.ConfigMap{
+			ObjectMeta: metaObj,
+			Data:       data,
+		}
+		configmap, err = k8sCore.CreateConfigMap(obj)
+		if err != nil {
+			return fmt.Errorf("failed to create configmap [%s] with data [%v]. Error : [%v]", configMapName, data, err)
+		}
+	} else {
+		log.InfoD("Updating Configmap: %v", configMapName)
+		configmap.Data = data
```

Comment:
This will overwrite the existing data with the latest data instead of updating it. We need to handle this by fetching the existing data and appending the new data to it, either at the test level or within method.

Reply:
Nice catch fixed.